tests/testAutopas/tests/autopasInterface/Newton3OnOffTest.cpp

```diff
@@ -5,121 +5,221 @@
-    autopas::MoleculeLJ defaultParticle;
-    RandomGenerator::fillWithParticles(*autoPas.getContainer(), defaultParticle, 100);
-    RandomGenerator::fillWithHaloParticles(*autoPas.getContainer(), defaultParticle,
-                                           autoPas.getContainer()->getCutoff(), 10);
-
-    EXPECT_CALL(mockFunctor, isRelevantForTuning()).Times(testing::AtLeast(1)).WillRepeatedly(Return(true));
-
-    // with newton 3:
-    int callsNewton3 = 0;
-    EXPECT_CALL(mockFunctor, allowsNewton3()).WillOnce(Return(true));
-    EXPECT_CALL(mockFunctor, allowsNonNewton3()).WillOnce(Return(false));
-    EXPECT_CALL(mockFunctor, AoSFunctor(_, _, true)).WillRepeatedly(testing::InvokeWithoutArgs([&]() {
-      callsNewton3++;
-    }));
-    EXPECT_CALL(mockFunctor, AoSFunctor(_, _, true)).Times(testing::AtLeast(1));
-    EXPECT_CALL(mockFunctor, AoSFunctor(_, _, false)).Times(0);  // disables newton3 variant
-    autoPas.iteratePairwise(&mockFunctor, autopas::DataLayoutOption::aos);
-
-    // without newton 3:
-    int callsNonNewton3 = 0;
-    EXPECT_CALL(mockFunctor, allowsNewton3()).WillOnce(Return(false));
-    EXPECT_CALL(mockFunctor, allowsNonNewton3()).WillOnce(Return(true));
-    EXPECT_CALL(mockFunctor, AoSFunctor(_, _, false)).WillRepeatedly(testing::InvokeWithoutArgs([&]() {
-      callsNonNewton3++;
-    }));
-    EXPECT_CALL(mockFunctor, AoSFunctor(_, _, false)).Times(testing::AtLeast(1));
-    EXPECT_CALL(mockFunctor, AoSFunctor(_, _, true)).Times(0);  // disables newton3 variant
-    autoPas.iteratePairwise(&mockFunctor, autopas::DataLayoutOption::aos);
-
-    EXPECT_EQ(callsNewton3 * 2, callsNonNewton3);  // should be called exactly two times
-
-    if (::testing::Test::HasFailure()) {
-      std::cerr << "Failures for containeroption: " << containerOption << std::endl;
-    }
-  }
-}
+              std::vector<std::string> ret;
 
-TEST_F(Newton3OnOffTest, testSoA) {
-  for (auto containerOption : autopas::allContainerOptions) {
-    if (containerOption == autopas::ContainerOptions::verletLists ||
-        containerOption == autopas::ContainerOptions::verletListsCells ||
-        containerOption == autopas::ContainerOptions::verletClusterLists) {
-      continue;
-    }
-    autoPas.init(getBoxMin(), getBoxMax(), getCutoff(), getVerletSkin(), getVerletRebuildFrequency(),
-                 {containerOption});
-    autopas::MoleculeLJ defaultParticle;
-    RandomGenerator::fillWithParticles(*autoPas.getContainer(), defaultParticle, 100);
-    RandomGenerator::fillWithHaloParticles(*autoPas.getContainer(), defaultParticle,
-                                           autoPas.getContainer()->getCutoff(), 10);
+              for (auto containerOption : autopas::allContainerOptions) {
+                // @TODO: let verlet lists support Newton 3
+                if (containerOption == autopas::ContainerOption::verletLists ||
+                    containerOption == autopas::ContainerOption::verletListsCells ||
```

Comment:
verletListsCells should actually support newton3...

Reply:
they should but will produce errors because of:
https://github.com/AutoPas/AutoPas/blob/880dd85c3457499d7d3af65c8b9bf86ac69a7c3f/src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h#L111-L114

Reply:
#173